### DISCUSSION TITLE
Put text next to a roi

### DISCUSSION THREAD

**Discussion by wasserth** [fury-gl/fury#658] [General]
Hello,
I am using fury to render 3D plots of binary masks. Across the entire image I have several binary masks at different positions. Now I want to show text next to each binary mask. I know the position of each mask in 3D space, but as far as I know text can only be plotted using `TextBlock2D` which can not take a position in 3D space. Now I am not sure how to find the right 2D position for the text from the 3D position of the binary mask.
Do you have a solution for this?

**Comment by skoudoro** [comment]
Hi @wasserth, 

You can use the 'container' actor or the gridUI to do that. 

I am surprised that we do not have a tutorial for that but you can look at the test of container actor or the grid : https://github.com/fury-gl/fury/blob/master/fury/tests/test_actors.py#L1104

Run the tests with interactive to true to see the results.

I hope it helps

**Comment by skoudoro** [comment]
Also,  there is `actor.vector_text` and `actor.text_3d(...)` if you want to setup the 3D position of a text actor in a simpler way

**Comment by wasserth** [comment]
Great! `actor.vector_text()` is what I was looking for.
However, when I am changing the camera a bit, then the labels are not facing the camera anymore. Is there a way how I can define the camera (what angle to look at the rendering) and still have the labels face towards the camera?
I am setting the camera like this:
```
showmanager.scene.set_camera(position=(-182., 109., 677.),
                            focal_point=(124., 109., 145.),
                            view_up=(0.0, 1.0, 0.0))
```

**Reply by skoudoro** [reply]
We are working on it to make it easier to follow the camera on #489

**Reply by skoudoro** [reply]
Hi @wasserth,

you might be interested by the following PR: https://github.com/fury-gl/fury/pull/661

we are adding follow the camera as an option

**Reply by wasserth** [reply]
Great. Thanks for the hint.
